### PR TITLE
Report Electron-resolved proxy to BQ on startup

### PR DESCRIPTION
**Pull request by ahmedalsudani**
@fjvallarino @tejohnso please review

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

File: installer/telemetry.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    mainWindow = _mainWindow;
+    module.exports.runProxyExperiment();
+  },
+  reportProxy(url, proxy) {
+    log.external("resolved proxy result", `${url}: ${proxy}`);
+  },
+  runProxyExperiment() {
+    const session = mainWindow.webContents.session;
+    const urls = ["risevision.com", "127.0.0.1", "192.168.1.1", "10.0.0.1"];
+    urls.map((url)=>{
```
Comment (tejohnso) [inline installer/telemetry.js]: @ahmedalsudani This maps the url array to a bunch of undefineds and then does nothing with the mapping.  I think forEach would be more appropriate.

**Comment by tejohnso** [comment]
@ahmedalsudani One comment to consider - looks good :+1:

File: installer/telemetry.js (hunk 1 above)
Comment (ahmedalsudani) [inline installer/telemetry.js]: @tejohnso ah, good point. Going to replace with a forEach before merging.

**Pull request merged**